common/src/main/java/com/pg85/otg/generator/resource/BoulderGen.java

```diff
@@ -38,6 +39,11 @@ public void spawn(LocalWorld world, Random random, boolean villageInChunk, int x
         if (y < this.minAltitude || y > this.maxAltitude) {
             return;
         }
+        
+        if (!bLoaded ) {
```

Comment:
Could put this in a method in Resource instead, no repeating code for each Gen class?